workspaces/diff-engine/package.json

```diff
@@ -14,6 +14,13 @@
     "url": "ssh://git@github.com/useoptic/optic-package.git",
     "directory": "workspaces/diff-engine"
   },
+  "files": [
+    ".env",
+    "scripts",
+    "lib",
+    "Cargo.toml",
+    "cli/Cargo.toml"
+  ],
```

Comment:
Diff-engine uses a `.npmignore`, that's why this wasn't in there. We had _multiple_ borked publishes because of this opt-in list instead of opting out. If that needs to be switched (I still wouldn't, but 🤷‍♂️), we should at least get rid of the other, as having both is way worse than choosing whichever one. 

Also, why does the npm package need cargo files published?

Reply:
@JaapRood uses the cargo.toml to extract the binary name